# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apcupsd/install.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- apcupsd-3.14.4/src/Makefile.vanilla	2008-05-26 10:24:11.000000000 +0200
+++ apcupsd-3.14.4/src/Makefile	2008-05-26 10:24:39.000000000 +0200
@@ -46,13 +46,13 @@
 
 install-base:
 	$(call MKDIR,$(sbindir))
-	$(call INSTPROG,700,apcupsd,$(sbindir)/apcupsd)
-	$(call INSTPROG,700,apctest,$(sbindir)/apctest)
-	$(call INSTPROG,755,apcaccess,$(sbindir)/apcaccess)
-	$(call INSTPROG,755,smtp,$(sbindir)/smtp)
+	$(call INSTPROG,700,apcupsd,$(sbindir)/)
+	$(call INSTPROG,700,apctest,$(sbindir)/)
+	$(call INSTPROG,755,apcaccess,$(sbindir)/)
+	$(call INSTPROG,755,smtp,$(sbindir)/)
 
 install-powerflute:
-	$(call INSTPROG,755,powerflute,$(sbindir)/powerflute)
+	$(call INSTPROG,755,powerflute,$(sbindir)/)
 
 uninstall-base:
 	$(call UNINST,$(sbindir)/apcupsd)
--- apcupsd-3.14.4/autoconf/targets.mak.vanilla	2008-05-26 10:28:00.000000000 +0200
+++ apcupsd-3.14.4/autoconf/targets.mak	2008-05-26 10:28:23.000000000 +0200
@@ -152,7 +152,7 @@
 # Install a program file, given mode, src, and dest
 define INSTPROG
    @echo "  COPY " $(2) =\> $(DESTDIR)$(3)
-   $(V)$(INSTALL_PROGRAM) $(STRIP) -m $(1) $(2) $(DESTDIR)$(3)
+   $(V)$(INSTALL_PROGRAM) -m $(1) $(2) $(DESTDIR)$(3)
 endef
 
 # Install a data file, given mode, src, and dest
